<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Apples-MBP-2" failures="1" tests="7" name="general.More_Module" time="358.933" errors="0" timestamp="2024-02-02T16:29:24 IST" skipped="0">
  <testcase classname="general.More_Module" name="Supplier" time="58.582"/>
  <system-out/>
  <testcase classname="general.More_Module" name="Purchase_Orders" time="36.616"/>
  <system-out/>
  <testcase classname="general.More_Module" name="Invoices" time="38.668"/>
  <system-out/>
  <testcase classname="general.More_Module" name="Quotes" time="38.826"/>
  <system-out/>
  <testcase classname="general.More_Module" name="Projects" time="62.649">
    <failure message="An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.141.59&#039;, revision: &#039;e82be7d358&#039;, time: &#039;2018-11-14T08:17:03&#039;
System info: host: &#039;Apples-MBP-2&#039;, ip: &#039;fe80:0:0:0:40f:bbdc:f93b:a90b%en0&#039;, os.name: &#039;Mac OS X&#039;, os.arch: &#039;x86_64&#039;, os.version: &#039;13.6.4&#039;, java.version: &#039;21.0.1&#039;
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/apple/Library/Develo..., appActivity: com.impossible-research.san..., appPackage: com.impossible-research.san..., autoAcceptAlerts: true, automationName: XCUITest, browserName: , databaseEnabled: false, deviceName: iPhone 15 Pro Max, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, takesScreenshot: true, udid: 2D8F97C4-F4B4-40FA-92EF-36F..., webStorageEnabled: false}
Session ID: 9077707e-99c1-4030-8cb7-8e5aa45e3c48
*** Element info: {Using=xpath, value=//XCUIElementTypeButton[@name=&quot;Save&quot;]}" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: An element could not be located on the page using the given search parameters.
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'Apples-MBP-2', ip: 'fe80:0:0:0:40f:bbdc:f93b:a90b%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '13.6.4', java.version: '21.0.1'
Driver info: io.appium.java_client.ios.IOSDriver
Capabilities {app: /Users/apple/Library/Develo..., appActivity: com.impossible-research.san..., appPackage: com.impossible-research.san..., autoAcceptAlerts: true, automationName: XCUITest, browserName: , databaseEnabled: false, deviceName: iPhone 15 Pro Max, javascriptEnabled: true, locationContextEnabled: false, networkConnectionEnabled: false, noReset: true, platform: MAC, platformName: iOS, takesScreenshot: true, udid: 2D8F97C4-F4B4-40FA-92EF-36F..., webStorageEnabled: false}
Session ID: 9077707e-99c1-4030-8cb7-8e5aa45e3c48
*** Element info: {Using=xpath, value=//XCUIElementTypeButton[@name="Save"]}
at org.testng.Assert.fail(Assert.java:110)
at general.More_Module.Projects(More_Module.java:419)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:135)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:673)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:220)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:945)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:193)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:808)
at org.testng.TestRunner.run(TestRunner.java:603)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:429)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:423)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:383)
at org.testng.SuiteRunner.run(SuiteRunner.java:326)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1249)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
at org.testng.TestNG.runSuites(TestNG.java:1092)
at org.testng.TestNG.run(TestNG.java:1060)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
    </failure>
  </testcase> <!-- Projects -->
  <system-out/>
  <testcase classname="general.More_Module" name="Reports" time="85.078"/>
  <system-out/>
  <testcase classname="general.More_Module" name="Expense_category" time="38.514"/>
  <system-out/>
</testsuite> <!-- general.More_Module -->
